### PR TITLE
Clean invalid utf-8 chars from subject

### PR DESCRIPTION
**Pull request by kelvinst**
Maybe now it's a little clearer. Right @gabebw. :wink:

### CODE REVIEW

File: spec/griddler/email_spec.rb [hunk 1]
```diff
@@ -639,6 +639,32 @@ def header_from_email(header)
...
+    expect(email.subject).to eq @subject
+  end
+
+  it 'handles invalid UTF-8 characters' do
+    email = Griddler::Email.new(
+      to: [@address],
+      from: @address,
+      subject: "\xc0\xc1\xf5\xfa\xfe\xff #{@subject}",
+    )
+    expected = "ÀÁõúþÿ #{@subject}"
```
Comment (kelvinst) [inline spec/griddler/email_spec.rb]: So @gabebw. This :point_up: is my problem.

**Comment by kelvinst** [comment]
@gabebw? Someone else?

**Comment by gabebw** [comment]
Merged -- thanks!

**Comment by kelvinst** [comment]
:tada: :fireworks: :shipit:

**Pull request merged**